game/operation/operation.py

```diff
@@ -90,6 +90,13 @@ def generate(self):
         if self.is_awacs_enabled:
             self.briefinggen.append_frequency("AWACS", "133 MHz AM")
 
+        # combined arms
+        self.mission.groundControl.pilot_can_control_vehicles = self.ca_slots > 0
+        if self.game.player == "USA":
```

Comment:
Can you please replace this with true coalition check instead?

Reply:
Do you have an example ?